README.md

```diff
@@ -10,11 +10,16 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versions
```

Comment:
So it would appear that gopkg.in is solving this by serving a git endpoint (which presumably just translates master <-> some release branch, I guess).